requirements_local

```diff
@@ -2,5 +2,5 @@ numpy==1.22.3
 pandas==2.0.3
 Pillow==9.0.0
 opencv-python==4.9.0.80
-tensorflow==2.9.0
-keras==2.9.0
+tensorflow==2.13.1
```

Comment:
what is the purpose of these version changes? if it is not a must, i will ask you not to change requirements.

Reply:
@serengil As I test it sir, when building the image, at pip install for tensorflow version 2.9.0, it can't find that version and it suggest for tensorflow version 2.13.1 sir.